src/modules/RA/DataLayer/components/tabs/PopupTab/PopupFieldList.js

```diff
@@ -69,21 +66,27 @@ const PopupFieldList = ({ fields, updateTemplate }) => {
     return errors;
   }, [translate]);
 
-  useEffect(() => {
+  const onTitleFieldChange = useCallback(cb => value => {
+    cb(value);
     updateTemplate();
   }, [updateTemplate]);
 
+  useEffect(() => {
+    updateTemplate();
+  }, [popupFields, updateTemplate]);
+
   return (
     <div className="wrapper" style={{ width: '50%' }}>
       <h4>{translate('datalayer.form.popup.title-field')}</h4>
       <Field
         name="popup_config.wizard.fields[0]"
         validate={validateTitle}
+        initialValue={useMemo(() => ({ sourceFieldId: mainFieldId }), [mainFieldId])}
```

Comment:
I don't think using a useMemo here is a good idea. Not really common nor recommended.

Reply:
Ok, I'll remove it

Reply:
I move it and assigned it to a variable in the component